comp/netflow/format/format.go

```diff
@@ -0,0 +1,138 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I merged them mainly because there were a lot of different linter errors in different files and I found it hard to track them down; since all of these were just a few lines long and they all did the same thing, I consolidated them into a single module (and renamed it to 'format' because every function was taking something and formatting it into a human-readable string). I can split those back out into individual files, that's fine.